paasta_tools/contrib/shared_ip_check.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from collections import defaultdict
+
+from paasta_tools.utils import get_docker_client
+
+
+def main():
+    docker_client = get_docker_client()
+    ip_to_containers = defaultdict(list)
+
+    for container in docker_client.containers():
```

Comment:
The docs for this version of docker-py is 404 now, does this return only running containers?

Reply:
Good point. Just tested it on a machine with stopped containers, looks like it does only return running containers.